.travis.yml

```diff
@@ -42,7 +42,7 @@ install:
   - 'bower update'
 
 before_script:
-  - grunt # this does syntax checking, unit testing, and jshint
+  - grunt --force # this does syntax checking, unit testing, and jshint
```

Comment:
Do not add a `--force` here. This may stop grunt from properly reporting syntax or semantic errors.